installer/manifests/keptn/values.yaml

```diff
@@ -512,6 +512,8 @@ bridge:
     mongoConnectionString: ""
   ## @param bridge.authMsg Overrides Keptn auth command displayed inside the Bridge user menu
   authMsg: ""
+  ## @param bridge.basicAuthUsername Keptn auth username for Bridge user login
+  basicAuthUsername: ""
```

Comment:
Please add the default value here too:
```suggestion
  basicAuthUsername: "keptn"
```
(don't forget to re-generate the helm value readme :) )

Reply:
Thanks again! I made it. 🛩️